docs/upgrade-guide-1.6-to-1.7.ipynb

```diff
@@ -65,11 +80,11 @@
     ]
    },
    "source": [
-    "### Update default 'admin' profile to prevent its deletion\n",
+    "### Update default `admin` profile to prevent its deletion\n",
     "\n",
-    "In Charmed Kubeflow 1.6 a special default profile named 'admin' is created at deployment time. When upgrading to 1.7 this default profile needs to be updated in order to prevent its deletion.\n",
+    "In Charmed Kubeflow 1.6 a special default profile named `admin` is created by default at deployment time. When upgrading to 1.7 this default profile needs to be edited in order to prevent its deletion.\n",
```

Comment:
```suggestion
    "In Charmed Kubeflow 1.6 a user profile named `admin` is created by default at deployment time.  This profile has no additional priviledges - it is just a default profile that was created for convenience and has been removed as of Charmed Kubeflow 1.7.  When upgrading to 1.7 this default profile will be deleted.  If you depend on this profile, you can do the following to prevent its deletion:\n",
```

Reply:
My guess is many people never knew this profile existed, and since it has no special powers most would be fine with it going away

Reply:
The problem is with out deployment guide. It calls for user `admin`. If our guide is followed then by default all work is in that `admin` namespace.